src/services/local-storage.service.js

```diff
@@ -4,7 +4,8 @@ export const clearLocalStorage = () => {
     _id: null,
     wishlist: null,
     language: 0,
-    cart: null
+    cart: null,
+    theme: true
```

Comment:
Why does theme have `boolean` value? I think it should have values like `LIGHT` or `DARK`

Reply:
I did this before, but LIGHT/DARK makes a lot of duplicate code

Reply:
Done